lib/getTinymanPrice.ts

```diff
@@ -14,26 +14,33 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-const axios = require("axios");
+import { Environment } from "./types/config";
+import { getAccountInfo } from "./helper";
 
 const getTinymanPrice = async (
   assetId: number,
-  environment: string
+  environment: Environment,
+  assetDecimals: number,
+  poolInfoAddr?: string
```

Comment:
Same here, why isn't it optional? Can't you figure out what it is at the beginning of the runLoop (or even earlier)?